de/grndb_network.py

```diff
@@ -0,0 +1,118 @@
+"""Contains functions for creating w matrix and directed graph for GRNdb dataset."""
+from os.path import join, dirname
+import networkx as nx
+import pandas as pd
+import numpy as np
+import matplotlib.patches as mpatches
+import matplotlib.lines as mlines
```

Comment:
Have you used `mlines` anywhere? if not, please remove this line.

Reply:
You're right, I removed it!